code-challenges.test.js

```diff
@@ -26,17 +24,52 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+//Write function named divideThree with the parameter/argument named num
 
+const divideThree = (num) => {
 
 
 
-// b) Create the function that makes the test pass.
+    
+//Write if statment if num is divided by three it is strictly equal to zero.
+    if(num % 3 === 0){
 
+//return a string interpolation stating num is divisible by three
+        return `${num} is divisible by three`
+
+//write an else if statement of num divided by three is not equal to zero
+    } else if(num % 3 !==0){
+
+// return with string interpolation stating num is not divisible by three
+        return `${num} is not divisible by three`
+// input else statement to output something is wrong if there is an error.
+    } else {
+        return "Something is wrong"
+    }
+}
+
+// a) Create a test with expect statements for each of the variables provided.
+
+//write test with the method "divideThree" to return is divisible by three.
+describe("divideThree", () => {
+    test ("returns is divisible by three", () => {
+        expect(divideThree().toEqual("is divisible by three"))
+    })
+})
+
+
+// b) Create the function that makes the test pass.
+//write the function diviethree to return is divisible by three.
+divideThree = () => {
+    return "is divisible by three"
+} //Not sure why I can't pass the test.*/
 
 
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+    
```

Comment:
I would like to see the test here. I think writing the test out first will give you the ideas you need to get ready to code the problem out.